File: B_PL_Codes/10-Comando_if/if01.c
/*
Lógica de programação
Prof. Jonatha Costa
Exercício resolvido:8
Escreva uma programa que leia uma nota e verifique se aprovado ou não,
considerando 7,0 como critério.
Versão 1: Código básico para passo-a-passo, sem blocos de código.

*****************************************************
Exercício : Reescreva o código utilizando outra lógica, estratégia ou comandos.
*****************************************************
*/

#include <stdio.h>
int main()
{
    float nota;
    // Ler dados
    printf("Informe uma nota:\n");
    scanf("%f", &nota);
    // Testar
    if (nota >= 7.0)
    // Exibir resultados
    {
        printf("Candidato aprovado!\n");
    }
    else
    {
        printf("Candidado não aprovado!\n");
    }
    return 0;
}

File: B_PL_Codes/10-Comando_if/inJava/if01.java

/*
Lógica de programação
Prof. Jonatha Costa
Exercício resolvido utilizando Java
Escreva uma programa que leia uma nota e verifique se aprovado ou não,
considerando 7,0 como critério.
Versão 1: Código básico para passo-a-passo, sem blocos de código.
*/
import java.util.Scanner;

public class if01 {
    public static void main(String[] args) {
        float nota;
        // Leitura de daos
        Scanner entrada_dado = new Scanner(System.in);
        System.out.println("Informe uma nota:");
        nota = entrada_dado.nextFloat();

        // Teste e resultados
        if (nota >= 7.0) {
            System.out.println("Candidato aprovado!");
        } else {
            System.out.println("Candidato não aprovado!");
        }

        entrada_dado.close();
    }
}
